src/api/spec/models/attrib_spec.rb

```diff
@@ -9,6 +9,18 @@
     it { expect(attribute.fullname).to eq("#{attribute.namespace}:#{attribute.name}") }
   end
 
+  context "#container" do
```

Comment:
What is the difference to test in line 24-43? That you don't assign the container but instantiate with the object?